### PR TITLE
Fixed Routing(partially) and added by_user field to the addData function in LogsController

### PR DESCRIPTION
**Pull request by JustinRijsdijk**
Fixed Routing(partially, only works for the LOGS page at this moment) and added by_user field to the addData function in LogsController, this was not being set, and did not have a default value. Alse I have set the default value of this column to NULL in the newest SQL dump.

### CODE REVIEW

File: SQL/time_log_system_tls.sql [hunk 1]
```diff
@@ -19,7 +19,7 @@ SET time_zone = "+00:00";
 /*!40101 SET NAMES utf8mb4 */;
 
 --
--- Database: `time_log_system_tls_production`
```
Comment (SanderT2001) [inline SQL/time_log_system_tls.sql]: Keep this the same.

File: SQL/time_log_system_tls.sql [hunk 2]
```diff
@@ -42,7 +43,9 @@ CREATE TABLE `clients` (
   `client_postal_code` varchar(35) DEFAULT NULL,
   `client_country` varchar(100) DEFAULT NULL,
   `client_description` text,
-  `by_user` int(11) NOT NULL
+  `by_user` int(11) NOT NULL,
+  PRIMARY KEY (`ID`),
+  KEY `client_postal_code` (`client_postal_code`)
```
Comment (SanderT2001) [inline SQL/time_log_system_tls.sql]: Unnecessary.

**Review by SanderT2001 — CHANGES_REQUESTED** [review]
See the comments in the PR.

File: SQL/time_log_system_tls.sql (hunk 2 above)
Comment (JustinRijsdijk) [reply SQL/time_log_system_tls.sql]: I've made it unnecessary, will be removed later today.

**Pull request closed without merging**